src/configuration/XcodebuildConfiguration.php

```diff
@@ -60,6 +61,10 @@ private function loadConfig() {
     public function buildCommand(array $flags) {
         $this->loadConfig();
 
+        if ($this->otherFlags) {
```

Comment:
si hacés `arc lint` sobre este proyecto estas cosas te las encuentra. arc lint over arc lint to get lint errors on your linters!